main.go

```diff
@@ -111,6 +104,8 @@ func generate(request plugin.CodeGeneratorRequest) (*plugin.CodeGeneratorRespons
 			default:
 				return nil, fmt.Errorf("unknown value '%s' for enum_as_int_or_string", v)
 			}
+		} else if k == "empty_schema" {
```

Comment:
Nit: naming of key. Additional_empty_schema or something like that? 

Fine to go ahead with name as is if you have strong feelings. I just found the name confusing but it made sense in code as the variable is well named

Reply:
I like it!